### PR TITLE
remove ActionView::Helpers::FormTagHelper from ActiveHashcash

### PR DESCRIPTION
**Pull request by rocket-turtle**

### CODE REVIEW

File: test/active_hashcash_test.rb [hunk 1]
```diff
@@ -38,4 +39,15 @@ def test_hashcash_bits
     ActiveHashcash::Stamp.parse("1:20:220623:test::MPWRGuN3itbd1NiQ:008").update!(ip_address: "127.0.0.1")
     assert_equal(ActiveHashcash.bits + 3, controller.hashcash_bits)
   end
+
+  def test_hashcash_hidden_field_tag
```
Comment (rocket-turtle) [inline test/active_hashcash_test.rb]: I'm not used to work wirh minitest. Please let me know if I should change something in the test

**Comment by alexisbernard** [comment]
Thanks, it's indeed better without including FormTagHelper.

I think we can skip the test. Most of the time testing the HTML output brings annoyance more than anything. That makes me think that a test with a real browser could be great. If someone has some experience that would be helpful. It's a very long time I didn't do that :)

**Comment by rocket-turtle** [comment]
Hi @alexisbernard,

thank you for your review. What do I need to do get this moved forward? Should I delete the changes in _test/active_hashcash_test.rb_?

I have no experiance with setting up real browser tests.
But we testet the change in our app and it worked well.

**Comment by alexisbernard** [comment]
Hi,

> What do I need to do get this moved forward? Should I delete the changes in test/active_hashcash_test.rb?

Yes.

> I have no experiance with setting up real browser tests. But we testet the change in our app and it worked well.

I was just writing down my thoughts. No need to merge.

**Comment by rocket-turtle** [comment]
Hi @alexisbernard,

I removed the spec would you mind releasing a new version?

**Comment by alexisbernard** [comment]
Thanks. Sure, the 0.3.2 has been released.

**Pull request merged**